resources/snap_packs/responses/snap_pack.yml

```diff
@@ -0,0 +1,68 @@
+application/json:
+  schema:
+    $ref: "../models/snap_pack.yml"
+  example:
+    id: ord_0d6a16a3fff6318ac8f8008dc1
+    description: April Campaign
+    to:
+      id: adr_bae820679f3f536b
+      description:
+      name: HARRY ZHANG
+      company:
+      phone:
+      email:
+      address_line1: 210 KING ST STE 6100
+      address_line2:
```

Comment:
This may be a dumb question: Why is the `STE 6100` not apart of `address_line2`?

Reply:
Good call. Tbh, I used the self mailer response as a reference and copied that over. https://github.com/lob/lob-openapi/blob/9683237ff15820849045a834c05b07685a04b4ed/resources/self_mailers/responses/self_mailer.yml#L15